OptimizelySDK.Tests/App.config

```diff
@@ -4,10 +4,31 @@
     that apply only to the Test project.
 -->
 <configuration>
-    <appSettings>
 
-    </appSettings>
+  <configSections>
+    <section name="optlySDKConfigSection"
+             type="OptimizelySDK.Config.OptimizelySDKConfigSection, OptimizelySDK, Version=3.2.0.0, Culture=neutral, PublicKeyToken=null" />
+  </configSections>
+  
+  <optlySDKConfigSection>
+  
+    <HttpProjectConfig sdkKey="43214321" 
+                       url="www.testurl.com" 
```

Comment:
This url is not needed because you have the url format

Reply:
I think this is not for actual HttpConfigManager testing, it's only to check, if the url what we set is readonly through property or not.